### DISCUSSION TITLE
Commerce 20241026 Equipment YAMAHA STAGEPAS 200

### DISCUSSION THREAD

**Discussion by Pighome7** [J-tour/J-tour.github.io#23] [General]
#22 Issue conversation
### 合作方式討論

> 由於「YAMAHA合作意願投票」獲得過半數同意，我們將進一步決定與YAMAHA的合作模式。

上週[忻妍](@xinyannn1 )跟芯瑄收到了兩種方案，需要大家提供意見。

1. 藝人將提供報價，一旦確認合作，將協助拍攝使用該產品的街頭表演reels影片。
2. 產品交換：不提供額外酬勞，藝人協助拍攝reels影片後，公司將贈送產品給藝人。此方案目前尚未確定是否可行，若藝人有意選擇，需待芯瑄向公司確認後方可決定。

**Comment by Pighome7** [comment]
我覺得第二點一定是最好，前提是如果芯瑄的公司同意，至少賺到一個好用的工具~

**Comment by Sean880128** [comment]
我也覺得第二點比較好，在我們器材還沒完整跟沒什麼名氣的情況下。

**Comment by xinyannn1** [comment]
希望能爭取到  2. 產品交換！
報價的部分，可以隨著日後合作經驗累積，慢慢建立我們自己的業配價位~

**Comment by ShengHaoLo** [comment]
不管選一還是二，都要先充實官網的資料，跟YAMAHA談判時才有籌碼

**Reply by ShengHaoLo** [reply]
#22 Shehbey commercial cooperation issue

**Comment by xinyannn1** [comment]
合作影片腳本投稿

**Reply by ShengHaoLo** [reply]
以下為腳本投稿模板開放使用，待[忻妍]( @xinyannn1 )詢問到更詳盡的供應商需求後，即可撰寫

**[腳本名稱]**

> [腳本大綱]

△ [場景]
▲ [角色]
▽ [內容]
▼ [參照]

**Reply by Sean880128** [reply]
[我愛學長經驗談]
>  畫面一  學弟在社團討論該選甚麼器材，大學長回學校打招呼，學生問他 : 學長聽說你有在外面表演，我們想問你該選甚麼音箱?

>  畫面二  學長就回憶表演的記憶，遇到過甚麼問題(當場被合作廠商或路人建議或謾罵的經驗)

>  畫面三  然後最後就在網路上找到yamaha的音箱 

>  畫面四  到學長介紹這個音箱(畫面是學長現場表演的畫面)
## 場景
社辦或校園場景
## 角色
一群學弟 (夢想當街頭藝人卻不知器材從何下手)
大學長 (畢業已經當街頭藝人一陣子的)
## 內容
學弟在社團討論該選甚麼器材，
大學長回學校打招呼，學生問學長，
學長回憶表演遇過的問題，
最後網路上找到yamaha的音箱 
學長介紹這個音箱                       [完] 
                                        
## 參照
有參照，但我忘記看到啥廣告的同樣的概念

**Reply by Pighome7** [reply]
場景
-
街頭表演場地

角色
-
一、介紹者
二、表演團體

內容
-
﹝影片開始﹞

【介紹者】對畫面提出詢問「街頭藝人除了樂器以外還需要什麼呢?那就是音響」
【介紹者】開始介紹此音響的功能性(PA、內建效果器、續航力、音質及擴音能力)
【表演團體】使用此音響進行團體表演示範(一小段)
【介紹者】說明以前需要帶不少器材(PA台、效果器、音響)會很麻煩，現在只要帶這顆音響就變得很便利

﹝影片結束﹞

參照
-
此為我自己的想法

**Comment by ShengHaoLo** [comment]
@xinyannn1 上次討論以互惠的方式執行，再請回報 YAMAHA最後希望我們怎麼配合

**Reply by ShengHaoLo** [reply]
已於 2024/10/26 完成reels影片拍攝，再請進行後續影片剪輯與文案寫作

@xinyannn1 請協助後續YAMAHA的交貨與請款流程，目前影片有需要協助的地方也請列出來 感謝
